cmd/analyzer/analyzer.go

```diff
@@ -97,6 +97,10 @@ func parseFlags(version string) cliFlags {
 	fs.StringVar(&cli.usageOpts.DailyUsageTable, "daily-data-table", "livepeer-analytics.staging.explorer_day_data", "BigQuery table to read total usage metrics from")
 	fs.StringVar(&cli.usageOpts.UsersTable, "users-table", "livepeer-analytics.staging.studio_users", "BigQuery table to read studio users from")
 	fs.Int64Var(&cli.viewsOpts.MaxBytesBilledPerBigQuery, "max-bytes-billed-per-big-query", 100*1024*1024 /* 100 MB */, "Max bytes billed configuration to use for the queries to BigQuery")
+	fs.StringVar(&cli.viewsOpts.ClickhouseOptions.Addr, "clickhouse-addr", "", "Address of Clickhouse Database")
+	fs.StringVar(&cli.viewsOpts.ClickhouseOptions.Database, "clickhouse-database", "", "Database name in Clickhouse")
+	fs.StringVar(&cli.viewsOpts.ClickhouseOptions.User, "clickhouse-user", "", "Clickhouse User")
+	fs.StringVar(&cli.viewsOpts.ClickhouseOptions.Password, "clickhouse-password", "", "Clickhouse Password")
```

Comment:
Would be great to get these configs in a single URL like `clickhouse://user:password@address.com/database`. If there's no such standard out there let's not invent it here tho